### PR TITLE
add Null reply assertion macro to rm_utils (RMUtil_AssertNullReply)

### PR DESCRIPTION
**Pull request by daTokenizer**

### CODE REVIEW

**Comment by daTokenizer** [comment]
@gkorland  are you guys planning on merging this code? if so when? :kissing_heart:

File: rmutil/test_util.h [hunk 1]
```diff
@@ -19,6 +19,8 @@
                 
 #define RMUtil_Assert(expr) if (!(expr)) { fprintf (stderr, "Assertion '%s' Failed\n", __STRING(expr)); return REDISMODULE_ERR; }
 
+#define RMUtil_AssertNullReply(rep) RMUtil_Assert(RedisModule_CreateStringFromCallReply(rep) == NULL)
```
Comment (MeirShpilraien) [inline rmutil/test_util.h]: Why not checking ReplyType == REDISMODULE_REPLY_NULL

Comment (daTokenizer) [reply rmutil/test_util.h]: in order to keep with the convention of a self-contained assert, just like `RMUtil_AssertReplyEquals` below

Comment (daTokenizer) [reply rmutil/test_util.h]: that said, added for compatibility sake

**Pull request merged**